### PR TITLE
Add pybullet simulation environment for UR5e with robotiq gripper 2f140

### PR DESCRIPTION
**Pull request by taochenshh**

### CODE REVIEW

File: src/airobot/robot/ur5e_pybullet.py [hunk 1]
```diff
@@ -0,0 +1,736 @@
...
+        Block the code to wait for the joint moving to the specified goal.
+        The goal can be a desired velocity(s) or a desired position(s).
+        Max waiting time is self.cfgs.TIMEOUT_LIMIT
+
+        Args:
+            goal (float or list): goal positions or velocities
+            joint_name (str): if it's none, all the actuated
+                joints are compared.
+                Otherwise, only the specified joint is compared
+            mode (str): 'pose' or 'vel'
```
Comment (anthonysimeonov) [inline src/airobot/robot/ur5e_pybullet.py]: docstring typo, mode (str): 'pose' >>> mode (str): 'pos'

Comment (taochenshh) [reply src/airobot/robot/ur5e_pybullet.py]: thanks. It has been corrected.

**Comment by taochenshh** [comment]
Self-collision can be enabled by passing `self_collision=True` when the robot instance is created. By default, collision detection is enabled between different dynamic moving bodies in Pybullet. We can control whether we want self-collision to be enabled or not. There are some weird behaviors when self-collision is enabled on the gripper. So when `self_collision=True`, only self-collision checking on the UR arm is enabled.

File: examples/ur5e/get_joint_info.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+import airobot as ar
+
+
+def main():
```
Comment (pulkitag) [inline examples/ur5e/get_joint_info.py]: @taochenshh please include doc-strings of the what the main fucntion does.

File: examples/ur5e/joint_position_control.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import time
+
+import airobot as ar
+
+
+def main():
```
Comment (pulkitag) [inline examples/ur5e/joint_position_control.py]: @taochenshh doc-strings

File: examples/ur5e/joint_velocity_control.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import math
+import time
+
+import airobot as ar
+
+
+def sin_wave(t, f, A):
```
Comment (pulkitag) [inline examples/ur5e/joint_velocity_control.py]: docstring

File: examples/ur5e/joint_velocity_control.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    return A * math.cos(2 * math.pi * f * t)
+
+
+def main():
```
Comment (pulkitag) [inline examples/ur5e/joint_velocity_control.py]: docstring

File: examples/ur5e/set_ee_pose.py [hunk 6]
```diff
@@ -0,0 +1,28 @@
+import time
+
+import airobot as ar
+
+
+def main():
```
Comment (pulkitag) [inline examples/ur5e/set_ee_pose.py]: docstring

File: src/airobot/arutil.py [hunk 7]
```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+def print_red(skk):
```
Comment (pulkitag) [inline src/airobot/arutil.py]: docstrings

File: src/airobot/robot/robot.py [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+    def __init__(self, cfgs):
+        """
+
+        Args:
+            cfgs (YACS CfgNode): configurations for the robot
+        """
+        self.cfgs = cfgs
+
+    def go_home(self):
```
Comment (pulkitag) [inline src/airobot/robot/robot.py]: docstrings for each function.

File: src/airobot/robot/ur5e_pybullet.py [hunk 9]
```diff
@@ -0,0 +1,721 @@
...
+    Class for the pybullet simulation environment
+        of a UR5e robot with a robotiq 2f140 gripper.
+    """
+
+    def __init__(self, cfgs, render=False, seed=None, self_collision=False):
+        """
+        Constructor for the pybullet simulation environment
+        of a UR5e robot with a robotiq 2f140 gripper
+
+        Args:
```
Comment (pulkitag) [inline src/airobot/robot/ur5e_pybullet.py]: @taochenshh what style guide are you using?

File: src/airobot/robot/ur5e_pybullet.py [hunk 10]
```diff
@@ -0,0 +1,721 @@
...
+            pos = p.getJointState(self.robot_id, jnt_id)[0]
+        return pos
+
+    def get_jvel(self, joint_name=None):
+        """
+        Return the joint velocity(ies)
+
+        Args:
+            joint_name: If it's None, it will return joint velocities
+                of all the actuated joints. Otherwise, it will
```
Comment (pulkitag) [inline src/airobot/robot/ur5e_pybullet.py]: all actuated joints -- means all the joints? If so, remove the word "actuated".

File: src/airobot/robot/ur5e_pybullet.py [hunk 11]
```diff
@@ -0,0 +1,721 @@
...
+        else:
+            jnt_id = self.jnt_to_id[joint_name]
+            vel = p.getJointState(self.robot_id, jnt_id)[1]
+        return vel
+
+    def get_jtorq(self, joint_name=None):
+        """
+        Return the joint torque(s) applied during the last stepSimulation
+        in VELOCITY_CONTROL and POSITION_CONTROL modes.
+        In TORQUE_CONTROL, the applied joint motor torque is
```
Comment (pulkitag) [inline src/airobot/robot/ur5e_pybullet.py]: grammar issues in doc-string. Pass through grammarly.

File: src/airobot/robot/ur5e_pybullet.py (hunk 10 above)
Comment (taochenshh) [reply src/airobot/robot/ur5e_pybullet.py]: Nope. There are some joints on the fingers that are underactuated. There is only one actuated joint on the gripper.

File: src/airobot/robot/ur5e_pybullet.py (hunk 9 above)
Comment (taochenshh) [reply src/airobot/robot/ur5e_pybullet.py]: Google Python style.

Comment (taochenshh) [reply src/airobot/robot/ur5e_pybullet.py]: The code passes Flask8

File: examples/ur5e/get_joint_info.py (hunk 2 above)
Comment (taochenshh) [reply examples/ur5e/get_joint_info.py]: commit 5ae7bdbb7995390de89fb7a93095fab0e41abe50

File: examples/ur5e/joint_position_control.py (hunk 3 above)
Comment (taochenshh) [reply examples/ur5e/joint_position_control.py]: commit 5ae7bdbb7995390de89fb7a93095fab0e41abe50

File: examples/ur5e/joint_velocity_control.py (hunk 4 above)
Comment (taochenshh) [reply examples/ur5e/joint_velocity_control.py]: commit 5ae7bdbb7995390de89fb7a93095fab0e41abe50

File: examples/ur5e/joint_velocity_control.py (hunk 5 above)
Comment (taochenshh) [reply examples/ur5e/joint_velocity_control.py]: commit 5ae7bdbb7995390de89fb7a93095fab0e41abe50

File: examples/ur5e/set_ee_pose.py (hunk 6 above)
Comment (taochenshh) [reply examples/ur5e/set_ee_pose.py]: commit 5ae7bdbb7995390de89fb7a93095fab0e41abe50

File: src/airobot/arutil.py (hunk 7 above)
Comment (taochenshh) [reply src/airobot/arutil.py]: commit 5ae7bdbb7995390de89fb7a93095fab0e41abe50

File: src/airobot/robot/ur5e_pybullet.py (hunk 11 above)
Comment (taochenshh) [reply src/airobot/robot/ur5e_pybullet.py]: commit 5ae7bdbb7995390de89fb7a93095fab0e41abe50

**Pull request closed without merging**